ajax/libs/Iframe-Height-Jquery-Plugin/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Iframe-Height-Jquery-Plugin",
+  "filename": "iframeheight.min.js",
+  "version": "1.2.5",
+  "author": "Ilker Guller <me@ilkerguller.com> (http://ilkerguller.com)",
+  "description": "This plugin can get contents of iframe and set height of iframe automatically. Also it has cross-domain fix",
+  "keywords": [
+    "jquery",
+    "plugin",
+    "iframe",
+    "height",
+    "resize"
```

Comment:
How about adding more keyword such as `auto`?
https://github.com/Sly777/Iframe-Height-Jquery-Plugin/blob/master/bower.json#L13
https://github.com/Sly777/Iframe-Height-Jquery-Plugin/blob/master/component.json#L13
Thank you.

Reply:
It looks fine to me because it does't help people find this lib, instead it will lead another similar lib jquery-iframe-auto-height. How about just following the package.json from the upstream?

Reply:
It looks fine to me because it does't help people find this lib, instead it will lead another similar lib jquery-iframe-auto-height. How about just following the package.json from the upstream?

Reply:
I respect your decision.